server_test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bytes"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"log"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// Creates a new file upload http request with params as cmd arguments.
+func newRequest(uri string, fileKeyName string, params map[string]string, path string) (*http.Request, error) {
```

Comment:
what is _fileKeyName_? Seems like it is always "file"?

Reply:
Yeah, not needed